CMakeLists.txt

```diff
@@ -49,27 +49,31 @@ macro(build_console_bridge)
   list(APPEND extra_cmake_args "-DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}")
 
   include(ExternalProject)
-  # Get console_bridge 0.4.1
-  ExternalProject_Add(console_bridge-0.4.1
-    URL https://github.com/ros/console_bridge/archive/0.4.1.tar.gz
-    URL_MD5 6bb0f640db1712cf2277e7923e7bab68
+  find_package(Patch REQUIRED)
+
+  # Get console_bridge 0.4.4
+  ExternalProject_Add(console_bridge-0.4.4
+    URL https://github.com/ros/console_bridge/archive/0.4.4.tar.gz
+    URL_MD5 341f42d1657409166f199f403244c9f0
     TIMEOUT 600
     CMAKE_ARGS
       -DCMAKE_INSTALL_PREFIX=${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}_install
       ${extra_cmake_args}
       -Wno-dev
+    PATCH_COMMAND
+    ${Patch_EXECUTABLE} -p1 -N < ${CMAKE_CURRENT_SOURCE_DIR}/add-target.patch
```

Comment:
Why not release a new patch version of `console_bridge` which already contains this fix?

Reply:
> Why not release a new patch version of `console_bridge` which already contains this fix?

Yeah, that will work too.

Reply:
All right, c8baf64 switches to upstream console_bridge 0.5.0.